### ISSUE TITLE
Some EVM nodes don't work in realtime mode

### ISSUE THREAD

**Issue by droserasprout** [dipdup-io/dipdup#953]
See EVM Networks page in docs.

**Comment by scottnoidea** [comment]
Workaround for this is to remove the `ws_url` from the node datasource config.  It works OK with just the rpc URL configured.

**Comment by droserasprout** [comment]
Yeah, some node providers just don't expose the WS endpoint, I don't know why. Maybe there's no bug on our end. WebSocket connection is only needed to save some RPS and reduce latency a bit. I'll keep this ticket open until all this stuff is documented.

**Comment by scottnoidea** [comment]
Correct, some don't expose it.  In my tests, I used Alchemy which does provide WS endpoints and found that I couldn't get any events to fire in real-time mode.  Dipdup appears to be using the WS endpoints from what I can gather.
